infrastructure/cicsk8s/galasa-prod/galasa-prod/cps-properties.yaml

```diff
@@ -0,0 +1,237 @@
+#
+# Copyright contributors to the Galasa project
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.inttests.location
+data:
+    value: https://development.galasa.dev/main/maven-repo/inttests/dev/galasa/dev.galasa.inttests.obr/0.33.0/dev.galasa.inttests.obr-0.33.0-testcatalog.json
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.inttests.obr
+data:
+    value: mvn:dev.galasa/dev.galasa.inttests.obr/0.33.0/obr 
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: resource.management.dead.heartbeat.timeout
+data:
+    value: "40"
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: request.type.CLI.prefix
+data:
+    value: C
+--- 
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: request.type.LOCAL.prefix
+data:
+    value: L
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: request.type.REQUEST.prefix
+data:
+    value: R
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: resource.management.finished.timeout
+data:
+    value: "40"
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.cics-prod.description
+data:
+    value: CICS TS Tests - Production (New)
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.inttests.description
+data:
+    value: Galasa Integration tests
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.inttests.repo
+data:
+    value: https://development.galasa.dev/main/maven-repo/inttests/
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.simbank.description
+data:
+    value: SimBank Tests
+---
+apiVersion: galasa-dev/v1alpha1
+kind: GalasaProperty
+metadata:
+    namespace: framework
+    name: test.stream.simbank.location
+data:
+    value: http://galasa-galasa-prod1.cicsk8s.hursley.ibm.com/testcatalog/galasasimbank
```

Comment:
This'll need to be changed to https://prod1-galasa-dev.cicsk8s.hursley.ibm.com/api/testcatalog/galasasimbank (not sure if it's being used but it would be worth using the right URL :))